en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/transport-security.md

```diff
@@ -1,26 +1,92 @@
 # Transport Security
```

Comment:
Let's add a small introduction to Transport Security explaining why it is useful to configure this.

Reply:
Fixed https://github.com/wso2/docs-apim/pull/4094/commits/a7a78ab7e99d07bf48da80110f3c3dc95d6e77e0